### PR TITLE
Update FeedbackPage.vue

### PR DESCRIPTION
**Pull request by skidi3**
Embedded a Bug Report form in Feedback tab on CC Search
#244

ccfeedback

---

Developer Certificate of Origin Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors. 1 Letterman Drive Suite D4700 San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

### CODE REVIEW

File: src/pages/FeedbackPage.vue [hunk 1]
```diff
@@ -22,20 +22,16 @@
...
-          <a href="mailto:support-search@creativecommons.org">
-            support-search@creativecommons.org
-          </a>
-          or
-          <a href="https://github.com/creativecommons/cccatalog-frontend/issues">
-            log the issue directly at Github
-          </a>.
-        </p>
-      </div>
+        <iframe src="https://docs.google.com/forms/d/e/1FAIpQLSfb_6yq2Md0v6S-XzsyT7p1QVhqr7MWHqInKdyYh4ReaWn4FQ/viewform?embedded=true"
```
Comment (brenoferreira) [inline src/pages/FeedbackPage.vue]: The link to the form is wrong. The one in #244 is different

Comment (skidi3) [reply src/pages/FeedbackPage.vue]: Updated, Sorry about that!

**Comment by akmadian** [comment]
@brenoferreira Definitely agree, I think putting them in a collapsible or tabs could be a good idea.

**Comment by annatuma** [comment]
@brenoferreira seeing it live, I agree with @akmadian's idea of tabs. @skidi3 are you comfortable making changes so that instead of one form after the other, there are instead two tabs on the page, one for Feedback and one for Report a bug?

Alternatively @brenoferreira we could make these two completely distinct pages, linked from the nap, so when a user hovers over Feedback they see either "Feedback" or "Report a Bug". Any thoughts on that approach instead?

**Comment by skidi3** [comment]
Totally agree with @akmadian  about tabs, that will look really clean. @annatuma Yes, I'm totally comfortable in making these changes.
Alternatively, If you agree,  I can also make two sections for 'Report a bug' and 'How can we improve' which on click will redirect to their google forms.  

ccsearch-new

**Comment by annatuma** [comment]
I prefer that the forms are embedded in our site, so if we do go with the design you proposed we'd need two new standalone pages to embed the forms in. With that in mind, I think tabs is the better choice for now, to keep it all on the existing feedback path. Thank you for making these changes!

**Comment by kgodey** [comment]
@skidi3 are you still working on this?

**Comment by akmadian** [comment]
I'm closing this PR because of inactivity, the issue has been freed up for someone else to work on.

**Comment by cognitive137** [comment]
Hello, can I work on this issue?

**Pull request closed without merging**